### DISCUSSION TITLE
Discrepancy in Transaction of Batch Number on ZkSync Explorer - Etherscan Explorer.

### DISCUSSION THREAD

**Discussion by a1111198** [Cyfrin/foundry-full-course-cu#1943] [Q&A]
I encountered a discrepancy while checking a transaction in ZkSync Explorer for batch 488237. Here are the details:

- **ZkSync Explorer Batch:** [Batch 488237](https://explorer.zksync.io/batch/488237)
- **Etherscan Transaction:** [Transaction on Etherscan](https://etherscan.io/tx/0xb09c199508574ed5b470e1934f48c8187337e28b5c40b5a1fba747432ef85558)

When I check the decoded input data for the transaction on Etherscan, it shows an input of `batchNumber uint64 488236`.

**Discrepancy:**

- **ZkSync Explorer Batch Number:** 488237
- **Etherscan Decoded Input Batch Number:** 488236
![11](https://github.com/Cyfrin/foundry-full-course-cu/assets/60999356/f9f49e94-b1a7-4f3d-a178-da7f193f0a9a)
![2](https://github.com/Cyfrin/foundry-full-course-cu/assets/60999356/7c061da7-80d2-4d6f-a1fc-a878fbe8fab9)

Where is this discrepancy coming from?

**Reason for Inquiry:**

I was trying to understand proto-danksharding from Patrick's video in the Foundry advanced course and encountered this issue. I want to know if this is a bug in the explorer or if there's a deeper concept that I need to understand.

**Additional Context:**

Any clarification on this matter would be greatly appreciated, as it will help me in understanding the workings of proto-danksharding and any potential impacts or issues in transaction processing.

Thank you!

**Comment by EngrPips** [comment]
I haven't seen that content myself, so I can't really be of help here, but I don't want to leave you hanging. Trust that @PatrickAlphaC  or @ciaranightingale  will attend to this as soon as they are available.

**Comment by PatrickAlphaC** [comment]
Good question... Let me check with zkSync.

**Comment by bxpana** [comment]
@a1111198 the `1.` values on Etherscan are the for the previous batch, but if you scroll down you'll see the matching batch number under `newBatchData`

Glad to see more people learning about ZKsync and hope you're having a good time!

**Reply by EngrPips** [reply]
Interesting, Thanks for the heads-up, sir.

**Reply by bxpana** [reply]
My pleasure 😁
